Krypt/Source/PKCS8.swift

```diff
@@ -12,10 +12,23 @@ public final class PKCS8 {
     guard
       let dataString = pem.unsafeUtf8cString,
       let pemCString = convert_pkcs1_to_pkcs8(dataString)
-    else {
-      return nil
+      else {
+        return nil
     }
 
     return String(cString: pemCString)
   }
+
+  public static func encrypt(_ pem: Data, password: String) -> String? {
+    guard !password.isEmpty else { return nil }
+    guard
+      let pemCString = pem.unsafeUtf8cString,
+      let passwordCString = Data(password.utf8).unsafeUtf8cString,
+      let encryptedPEMCString = pkcs8_encrypt(pemCString, passwordCString)
```

Comment:
do we need to release cStrings?

Reply:
I'm not sure tbh: http://www.alwaysrightinstitute.com/swift-cstr/

Reply:
Yes we should probably use `.withCString(<#T##body: (UnsafePointer<Int8>) throws -> Result##(UnsafePointer<Int8>) throws -> Result#>)` https://www.uraimo.com/2016/04/07/swift-and-c-everything-you-need-to-know/#working-with-strings

Reply:
But again, should we go into this right now? Probably not, because we can spend days just optimising our C code.

Reply:
I agree. Let it be.